packages/x-date-pickers/src/DigitalClock/DigitalClock.tsx

```diff
@@ -294,6 +296,42 @@ export const DigitalClock = React.forwardRef(function DigitalClock<TDate extends
     utils.isEqual(option, valueOrReferenceDate),
   );
 
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    switch (event.key) {
+      case 'PageUp': {
+        if (!listRef.current) {
+          return;
+        }
```

Comment:
This code is impossible to reach right?

```suggestion
```

Reply:
> In theory listRef.current can be null

Only if the ref is not applied to the right element. I think that using `listRef.current!` so it gets transpiled to `listRef.current` is much better. No extra bundle size or runtime time spent on this. If done at scale, I would hope this brings more than a +1% improvement.

Reply:
That was my suggestion to go the safer route and prevent cases where DOM could be tampered with.
But given most other cases, where `<x>Ref.current` is accessed, we could indeed simplify to `listRef.current!`.